go.mod

```diff
@@ -190,3 +191,5 @@ replace github.com/Azure/go-autorest/autorest => github.com/Azure/go-autorest/au
 replace github.com/Azure/go-autorest/autorest/adal => github.com/Azure/go-autorest/autorest/adal v0.5.0
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.2.0+incompatible
+
+replace github.com/istio.io/proxy/src/envoy/tcp/metadata_exchange/config v0.0.0 => ./pilot/pkg/metadata_exchange
```

Comment:
Why we are doing this? Either import from pilot directly or import from istio.io/proxy?

If this is actually intended, and not just a WIP like previously indicated, can we at least comment why since its not intuitive at all